fire/cli/gnss/bernese.py

```diff
@@ -0,0 +1,378 @@
+import getpass
+from typing import List
+from io import BytesIO
+from zipfile import ZipFile
+
+import click
+from rich.table import Table
+from rich.console import Console
+from rich import box
+from sqlalchemy.exc import NoResultFound, IntegrityError
+
+import fire.cli
+from fire.cli import firedb
+from fire.io.bernese import BerneseSolution
+from fire.api.model import (
+    GNSSTidsserie,
+    Koordinat,
+    Beregning,
+    Observation,
+    ObservationsLængde,
+    KoordinatKovarians,
+    ResidualKovarians,
+    Punkt,
+    Srid,
+)
+from fire.cli.niv import bekræft
+from . import gnss
+
+SRIDINDEX = {
+    "IGb08": "EPSG:9015",
+    "IGS14": "EPSG:8227",
+}
+
+
+def find_relevante_punkter(
+    solution: BerneseSolution,
```

Comment:
Bemærker lige, at vi ikke taler dansk her. Skulle vi ikke kalde det løsning i stedet for solution?

Reply:
Jo, det skal vi måske. Men out-of-scope her.